netbox/dcim/views.py

```diff
@@ -341,8 +342,21 @@ def get(self, request):
             'reverse': reverse,
             'rack_face': rack_face,
             'filter_form': forms.RackElevationFilterForm(request.GET),
+            'show_images': request.user.config.get('rack_elevation.show_images', False)
         })
 
+    def post(self, request):
```

Comment:
Any such dynamic updates would need to use the REST API. (Adding individual view methods for all of the various settings does not scale.) Unfortunately, we don't yet have a REST API endpoint for user preferences. I believe this work is blocked until one can be introduced.

Reply:
Has a suitable problem been created or do I need to create a new one?

Reply:
The upcoming v2.10 release introduces an endpoint for this. If you want, you can rebase this PR off the `develop-2.10` branch and adapt to use the new endpoint, or just wait for the release (which should be sometime next week).

Reply:
OK, I'll do something about it soon.

Reply:
Do you have an update for this?